src/compose/paths.c

```diff
@@ -61,7 +61,7 @@ resolve_name(const char *filename, enum resolve_name_direction direction,
 
     xlocaledir = get_xlocaledir_path();
 
-    ret = snprintf(path, sizeof(path), "%s/%s", xlocaledir, filename);
+    ret = snprintf(path, sizeof(path), "%s" PATHSEPS "%s", xlocaledir, filename);
```

Comment:
According to the [Windows `fopen` documentation](https://docs.microsoft.com/en-us/previous-versions/yeby3zcb(v%3Dvs.140)#remarks), it does accept forward slash as path separator. Is it not so?

Reply:
True! I missed that part of the documentation, and I will be more than happy to see the hacky commit that tried to use the backslashes as path separators go away :smile: